src/api/utils/api_utils.py

```diff
@@ -267,20 +267,20 @@ def prepare_menu_items_for_portal(content, prefix):
             
         return prepared_menu_items
 
-def remove_unpublished_items(content):
+def remove_unpublished_items(content, links_to_hide=[]):
```

Comment:
Can we make the name of this function less generic, since it's only used to filter out menu items